app/src/main/java/org/oppia/android/app/player/exploration/ExplorationActivityPresenter.kt

```diff
@@ -104,6 +103,11 @@ class ExplorationActivityPresenter @Inject constructor(
       getExplorationFragment()?.handlePlayAudio()
     }
 
+    binding.actionBottomSheetOptionsMenu.setOnClickListener {
+      val bottomSheetOptionsMenu = BottomSheetOptionsMenu()
+      bottomSheetOptionsMenu.show(activity.supportFragmentManager, bottomSheetOptionsMenu.tag)
```

Comment:
Is there a showNow that can be used, instead?

Reply:
Done